lib/diagnostic.rb

```diff
@@ -43,6 +47,14 @@ def transform(words)
   # replacing the trailing newline character with a
   # space and return the result.
   def read_file(filename)
-    # Your code goes here
+    lines = File.open(ARGV[0]) do |file|
```

Comment:
We're given the filename as an argument, so there's no need to grab is as `ARGV[0]`.